doc/tutorial/network_ring.rst

```diff
@@ -0,0 +1,261 @@
+
+.. code-block:: python
+
+   # (2) Mark location for synapse at the midpoint of branch 1 (the first dendrite).
+   labels['synapse_site'] = '(location 1 0.5)'
+   # Mark the root of the tree.
+   labels['root'] = '(root)'
+
+After we've created a basic :py:class:`arbor.decor`, step **(3)** places a synapse with an exponential decay (``'expsyn'``) is on the ``'synapse_site'``.
+Note that mechanisms can be initialized with their name; ``'expsyn'`` is short for ``arbor.mechanism('expsyn')``.
+
+Step **(4)** places a spike detector at the ``'root'``. :py:class:`spike_detectors <spike_detector>` will send spikes into an
+:class:`arbor.connection`, whereas the :ref:`expsyn mechanism <mechanisms_builtins>` can receive spikes from an
+:class:`arbor.connection`.
+
+.. code-block:: python
+
+   decor = arbor.decor()
+
+   # Put hh dynamics on soma, and passive properties on the dendrites.
+   decor.paint('"soma"', 'hh')
+   decor.paint('"dend"', 'pas')
+
+   # (3) Attach a single synapse.
+   decor.place('"synapse_site"', 'expsyn')
+
+   # (4) Attach a spike detector with threshold of -10 mV.
+   decor.place('"root"', arbor.spike_detector(-10))
+
+   cell = arbor.cable_cell(tree, labels, decor)
+
+The recipe
+**********
+
+To create a model with multiple connected cells, we need to use a :py:class:`recipe <arbor.recipe>`.
+The recipe is where the different cells and the :ref:`connections <interconnectivity>` between them are defined.
+
+Step **(5)** shows a class definition for a recipe with multiple cells. Instantiating the class requires the desired
+number of cells as input. Compared to the :ref:`simple cell recipe <tutorialsinglecellrecipe>`, the main differences
+are connecting the cells, returning a configurable number of cells **(6)** and returning a new cell per ``gid`` **(7)**
+(``make_cable_cell()`` returns the cell above).
+
+Step **(8)** creates an :py:class:`arbor.connection` between this cell and the previous. (The ``gid`` of the previous
+cell is ``(gid-1)%self.ncells``.) The connection has a weight of 0.1 μS and a delay of 5 ms. The first two arguments
+to :py:class:`arbor.connection` are the **source** and **target** of the connection, and these are defined by the
+cell index ``gid`` and the source or target index. (:term:`Remember <connection>` that sources and targets are
+separately indexed.)
+
+The two endpoints are of type :class:`arbor.cell_member`, and can be initialized with a ``(gid,index)`` tuple.
+The cells have one synapse (step **3**), so the target endpoint has the 0th index. The cell has one
+spike generator (step **4**), so its source index is also 0.
+
+Lastly, we must inform the recipe how many connections we have on each cell (``gid``).
+:func:`arbor.cable_cell.num_targets` and :func:`arbor.cable_cell.num_sources` must be set to 1: each cell has one
+source and one target endpoint.
+
+Step **(9)** attaches an :py:class:`arbor.event_generator` on the 0th target (synapse) on the 0th cell; it is
```

Comment:
instead of `; it is connected to the ..`, I would say `this means it is connected to the ...`